yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -154,11 +155,55 @@ func prepareImportDataStatusTable() ([]*tableMigStatusOutputRow, error) {
 		}
 	}
 
+	renamedRowsForPartitions := make(map[string]*tableMigStatusOutputRow, 0)
+
 	for _, dataFile := range dataFileDescriptor.DataFileList {
 		row, err := prepareRowWithDatafile(dataFile, state)
 		if err != nil {
 			return nil, fmt.Errorf("prepare row with datafile: %w", err)
 		}
+		renamedTable := renameTableIfRequired(row.tableName)
```

Comment:
Not exactly sure what is happening here.. I thought we already modified the import data status related functions?

Reply:
No this is the only place where I am modifying the import data status related functions.

Reply:
I am doing the same thing I did at other places clubbing all the leaf partition rows to root table one. Getting all the leaf partitions, clubbing the count of imported leaf partitions for that root, etc... 
Let me know if you to connect to discuss this.